tests/wdio.conf.ts

```diff
@@ -0,0 +1,310 @@
+import { env } from "node:process";
+export const config: WebdriverIO.Config = {
+  //
+  // ====================
+  // Runner Configuration
+  // ====================
+  // WebdriverIO supports running e2e tests as well as unit and component tests.
+  runner: "local",
+  tsConfigPath: "./tsconfig.json",
+
+  //
+  // ==================
+  // Specify Test Files
+  // ==================
+  // Define which test specs should run. The pattern is relative to the directory
+  // of the configuration file being run.
+  //
+  // The specs are defined as an array of spec files (optionally using wildcards
+  // that will be expanded). The test for each spec file will be run in a separate
+  // worker process. In order to have a group of spec files run in the same worker
+  // process simply enclose them in an array within the specs array.
+  //
+  // The path of the spec files will be resolved relative from the directory of
+  // of the config file unless it's absolute.
+  //
+  specs: ["./src/**/*.test.ts"],
+  // Patterns to exclude.
+  exclude: [
+    // 'path/to/excluded/files'
+  ],
+  //
+  // ============
+  // Capabilities
+  // ============
+  // Define your capabilities here. WebdriverIO can run multiple capabilities at the same
+  // time. Depending on the number of capabilities, WebdriverIO launches several test
+  // sessions. Within your capabilities you can overwrite the spec and exclude options in
+  // order to group specific specs to a specific capability.
+  //
+  // First, you can define how many instances should be started at the same time. Let's
+  // say you have 3 different capabilities (Chrome, Firefox, and Safari) and you have
+  // set maxInstances to 1; wdio will spawn 3 processes. Therefore, if you have 10 spec
+  // files and you set maxInstances to 10, all spec files will get tested at the same time
+  // and 30 processes will get spawned. The property handles how many capabilities
+  // from the same test should run tests.
+  //
+  maxInstances: 10,
+  //
+  // If you have trouble getting all important capabilities together, check out the
+  // Sauce Labs platform configurator - a great tool to configure your capabilities:
+  // https://saucelabs.com/platform/platform-configurator
+  //
+  capabilities: [
+    {
+      // capabilities for local browser web tests
+      browserName: "chrome",
+      "goog:chromeOptions": {
+        // comment this to see the evaluation
+        args: ["headless", "disable-gpu"],
+      },
+    },
+  ],
+
+  //
+  // ===================
+  // Test Configurations
+  // ===================
+  // Define all options that are relevant for the WebdriverIO instance here
+  //
+  // Level of logging verbosity: trace | debug | info | warn | error | silent
+  logLevel: "info",
```

Comment:
I tested this locally and when I change this to `warn`, I get very useful output:

```suggestion
  logLevel: "warn",
```

Introducing a small error results in output like this:

```
% TEST_BOOK_DIR=../book/en/html npm run wdio

> wdio
> wdio run ./wdio.conf.ts


Execution of 2 workers started at 2024-11-28T15:58:10.041Z

[0-0] RUNNING in chrome - file:///src/basic.test.ts
[0-1] RUNNING in chrome - file:///src/speaker-notes.test.ts
[0-0] Error in "Basic test.should have the default_theme light"
Error: expect(received).toBe(expected) // Object.is equality

Expected: "xlight"
Received: "light"
    at Context.<anonymous> (.../src/comprehensive-rust/tests/src/basic.test.ts:7:63)
[0-0] FAILED in chrome - file:///src/basic.test.ts
[0-1] 2024-11-28T15:58:16.536Z WARN webdriver: WebDriverError: javascript error: {"status":60,"value":"[object HTMLButtonElement] has no size and location"}
[0-1]   (Session info: chrome=131.0.6778.85) when running "actions" with method "POST" and args "{"actions":[{"id":"action1","type":"wheel","parameters":{},"actions":[{"type":"scroll","x":-751,"y":1079,"deltaX":0,"deltaY":0,"duration":0,"origin":{"element-6066-11e4-a52e-4f735466cecf":"f.82DFDD24A66D5F65A979257D82375039.d.48683DE2EE1A6FB6FAD0CDAB4CA1C1B3.e.44"}}]}]}"
[0-1] PASSED in chrome - file:///src/speaker-notes.test.ts

Spec Files:	 1 passed, 1 failed, 2 total (100% completed) in 00:00:06  
```

Reply:
sounds good to me!